services/src/main/scala/cromwell/pipeline/service/ProjectConfigurationService.scala

```diff
@@ -72,9 +110,9 @@ object ProjectConfigurationService {
         } yield eitherFile
 
         val configurationVersion =
-          getLastByProjectId(projectId, userId).map {
-            case Some(configuration) => configuration.version.increaseValue
-            case None                => ProjectConfigurationVersion.defaultVersion
+          getLastOptionByProjectId(projectId, userId).transform {
+            case Success(Some(configuration)) => Success(configuration.version.increaseValue)
+            case _                            => Success(ProjectConfigurationVersion.defaultVersion)
```

Comment:
I don't think we should swallow an error and return the default version like nothing has happened